source/agora/node/Runner.d

```diff
@@ -96,6 +96,10 @@ public NodeListenerTuple runNode (Config config)
 
     setTimer(0.seconds, &node.start, Periodic.No);  // asynchronous
 
+    if (config.validator.enabled && config.admin.enabled)
+        // schedule
+        setTimer(0.seconds, &((cast(Validator)node).admin_start), Periodic.No);
+
```

Comment:
99% of casts in D are a mistake, because the type system is powerful enough that you don't need them.
In this case, one could trivially move the `setTimer` call in the `if` clause and use `inst` which is of type `Validator`.

But an even better approach is to just merge `admin_start` with `start` and remove this all together.